src/parser/types.ts

```diff
@@ -516,9 +638,31 @@ interface RandomInterruption {
   components: (string)[];
 }
 
+/**  The InterruptionBlock interface is used to define interruptions in a block. These can be used for breaks or attention checks. Interruptions can be deterministic or random. */
 export type InterruptionBlock = DeterministicInterruption | RandomInterruption;
 
-/** The IndividualComponentSingleResponseCondition interface is used to define a SkipCondition based on a single answer to a specific component. If the component is repeated within the block, this condition will only check the first instance of the component once the order is flattened. */
+/** The IndividualComponentSingleResponseCondition interface is used to define a SkipCondition based on a single answer to a specific component. If the component is repeated within the block, this condition will only check the first instance of the component once the order is flattened.
+ *
+ * For example, if you want to skip to a different component based on the a response to a specific component, you would use the IndividualComponentSingleResponseCondition. Here's an example of how to use the IndividualComponentSingleResponseCondition:
+ *
+ * ```js
+ * {
+ *   ...
+ *   "skip": [
+ *     {
+ *       "name": "attentionCheck",
+ *       "check": "response",
+ *       "responseId": "attentionCheckResponse",
+ *       "value": "the right answer",
+ *       "to": "end"
+ *     }
+ *   ]
+ *   ...
+ * }
+ * ```
+ *
+ * In this example, if the attentionCheckResponse response to the component with the id attentionCheck is "the right answer", the participant will be redirected to the end of the study. If the response is not equal to "the right answer", the participant will continue to the next component in the sequence.
```

Comment:
```suggestion
 * In this example, we assign our skip logic to the component whose ID is "attentionCheck". If the answer given to the response "attentionCheckResponse" is equal to "the right answer", then the user will be redirected to the end of the study. If the response is _not_ equal to "the right answer", then the participant will continue to the next component in the sequence. 
```
Tried to separate this sentence a bit to make it easier to read